src/jit/regalloc.h

```diff
@@ -5,8 +5,6 @@
 #ifndef REGALLOC_H_
 #define REGALLOC_H_
 
-// Some things that are used by both LSRA and regpredict allocators.
-
 enum FrameType
```

Comment:
For clarity, it would probably be good to have a comment that explains why this is separate from the lsra files, and eventually we should consider restructuring - though perhaps we should keep the regalloc files as "common" functionality in the event that we add an alternate register allocator in future.

Reply:
Even if we don't have two register allocators, it doesn't seem a bad thing to keep the things that are interfaces to the rest of the compiler separate, but it's not like we have a clean separation now. I'd probably leave the regalloc stuff alone and perhaps even migrate some other things there over time.
However, for codegenlinear, I agree that it would be better moved to codegencommon.cpp and codegen.h. I don't feel it's urgent, though.